tests/app/test_jinja_filters.py

```diff
@@ -158,6 +170,36 @@ def test_format_conditional_date(self):
             # Then
             self.assertEqual(format_value, "<span class='date'>{date}</span>".format(date=triple[2]))
 
+    def test_calculate_years_difference(self):
+        # Given
+        ten_years_ago = (datetime.today()+relativedelta(years=-10)).strftime('%Y-%m-%d')
+
+        date_list = [('2017-01-30', '2018-01-30', '1 year'),
+                     ('2015-02-02', '2018-02-01', '2 years'),
+                     ('2016-02-29', '2017-02-28', '1 year'),
+                     ('2016-02-29', '2020-02-28', '3 years'),
+                     (ten_years_ago, 'now', '10 years')]
+
+        for dates in date_list:
+            start_date = dates[0]
+            end_date = dates[1]
+
+            # When
+            calculated_values = calculate_years_difference(start_date, end_date)
```

Comment:
minor, but this should be `calculated_value`. plural implies that it is a list

Reply:
Was a list at one point. Whoops.